controller/pid_parameters.py

```diff
@@ -3,14 +3,15 @@
 class PIDParameters:
 
     def __init__(self):
```

Comment:
The parameters should be set from the config file and since the PIDParameters class should be unaware of the config implementation I would pass all values as function parameters